### PR TITLE
Add Chinese Language

### PR DESCRIPTION
**Pull request by 239573049**
Fixes #

## Proposed Changes

- subscriptions/languages/zh-CN/spec.md
- subscriptions/spec.md
- docs/languages/zh-CN/SDK-maintainer-guidelines.md
- docs/SDK-maintainer-guidelines.md
- docs/languages/zh-CN/SDK-PR-guidelines.md
- docs/SDK-PR-guidelines.md
- docs/languages/zh-CN/README.md
- docs/README.md
- README.zh-cn.md
- README.md

**Release Note**

```release-note
Adding Chinese version of README 
```

### CODE REVIEW

**Comment by duglin** [comment]
@239573049 can you please sign your commit?

**Comment by duglin** [comment]
@239573049 I think the "verify" errors are real. Please take a look.

@JieDing can you review?

**Comment by JieDing** [comment]
> @239573049 I think the "verify" errors are real. Please take a look.
> 
> @JieDing can you review?

Ok, I'm on it.

**Comment by 239573049** [comment]
I tried to fix it

**Comment by 239573049** [comment]
How do I trigger validation again

**Comment by duglin** [comment]
Did you push a new commit?

**Comment by 239573049** [comment]
yes

**Comment by duglin** [comment]
I don't see any new commits but I reran the checks again and they still fail

**Comment by 239573049** [comment]
how about now?

**Comment by JieDing** [comment]
@239573049  I went through the PR briefly, and I saw some of your proposed documents violate the doc structure of translations. Please read the [translation spec](https://github.com/cloudevents/spec/blob/main/languages/languages-spec.md) and reorg your files. Thx a lot.

File: languages/zh-CN/README.md [hunk 1]
```diff
@@ -1,131 +1,27 @@
...
+- [贡献者](/docs/contributors.md):对
+  CloudEvents 规范和生态系统
 
-对事件的统一描述的匮乏意味着开发者必须不断重新学习如何消费不同定义的事件。
-它同样限制了那些用来帮助事件数据完成跨环境传输的库(如 SDKs)、工具(如事件路由器)和基础设施(如事件追踪系统)的发展。
-总体来看，这种匮乏严重阻碍了事件数据的可移植性和生产力。
+- [Open Source](/docs/open-source.md):支持 CloudEvents 的开源库
 
-CloudEvents 是一个以通用格式来描述事件数据的规范。它提供了事件在服务、平台和系统中的互操作性。
+- [Demos](/docs/demos.md): CloudEvents in action
```
Comment (duglin) [inline languages/zh-CN/README.md:14]: Pretty sure these need to be relative links. See the GitHub checker output

**Comment by duglin** [comment]
Please sign your commmits. The DCO checker isn't happy

**Comment by duglin** [comment]
@239573049 any updates on this?

**Comment by 239573049** [comment]
> @239573049 any updates on this?

I want to know how to write the relative route?

**Comment by duglin** [comment]
See example in the existing docs, for example: https://github.com/cloudevents/spec/pull/1117/files#diff-ce92ed2e6f8227bc027a7236fae2286a3bce9ceafe4d85e99872ba44c9f0bd7cL83

**Comment by 239573049** [comment]
Signed-off-by: 贺家乐 239573049@qq.com

**Comment by 239573049** [comment]
@duglin I wonder if I can continue to verify it

**Comment by duglin** [comment]
you'll need to rebase on the last version, there are merge conflicts

**Comment by duglin** [comment]
you'll also need to sign all of your commits. It would probably be easier if you just squash it down to one commit while you're rebasing

**Comment by 239573049** [comment]
> you'll also need to sign all of your commits. It would probably be easier if you just squash it down to one commit while you're rebasing

ok

**Comment by 239573049** [comment]
> you'll also need to sign all of your commits. It would probably be easier if you just squash it down to one commit while you're rebasing

So let me close this PR

**Pull request closed without merging**